pygsheets/client.py

```diff
@@ -55,12 +57,16 @@ def __init__(self, credentials, retries=3, http=None, check=True, seconds_per_qu
         self.oauth = credentials
         self.logger = logging.getLogger(__name__)
 
-        http = AuthorizedHttp(credentials, http=http)
+        http = AuthorizedHttp(credentials, http=httplib2.Http())
```

Comment:
this means we are not using the user-passed `http` param anymore, so it is a breaking change. i think this should be `http or httplib2.Http()`

Reply:
I've created a conditional for that.